lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +210,39 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self.seed_urls = config.get_seed_urls()
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        return article_bs.get('href')
```

Comment:
who guarantees that there will be a link attribute? you potentially return None, later ignore it, also breaking interface that requires you to always return a string